images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -20,19 +20,36 @@ function Get-FortranVersions {
     return "GNU Fortran " + ($fortranVersions -Join ", ")
 }
 
-function Get-ClangVersions {
-    $clangVersions = @()
+function Get-ClangTool {
```

Comment:
I think `Get-ClangToolVersion` would be more clear name

Reply:
Agree, thanks!